index.html

```diff
@@ -3,9 +3,147 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <title>DesignX Company - Conference Web Page</title>
 </head>
 <body>
-    
+<!--HEADER-->
+    <div class="background">
+        <header class="header">
+            <nav class="navbar">
+                <a href="index.html" class="nav-branding"><img src="images/dark logo.png" alt="logo"></a>
+
+                <ul class="nav-menu">
+                    <li class="nav-item">
+                        <a href="about.html" class="nav-link orange-text">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#program" class="nav-link">Program</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#featured" class="nav-link">Featured</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#sponsor" class="nav-link">Sponsor</a>
+                    </li>
```

Comment:
- You have done well so far, but your navbar is not styled as it is in the design template. There is a missing top-dark bar in your project. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please fix this for both the index and about page 👍 👍 
Figma design:
<img width="705" alt="wgrgwerg" src="https://github.com/78fatemanazari/Capstone-project/assets/73607512/7a5210cf-8747-44c4-97a4-02bd63315612">

Your project:
<img width="948" alt="ererge" src="https://github.com/78fatemanazari/Capstone-project/assets/73607512/1814648c-c990-4c65-b382-e6e3336cb1c8">